pkgs/by-name/pc/pcsx2/package.nix

```diff
@@ -144,9 +16,23 @@ llvmPackages_17.stdenv.mkDerivation (finalAttrs: {
       PC, with many additional features and benefits.
     '';
     homepage = "https://pcsx2.net";
-    license = with licenses; [ gpl3Plus lgpl3Plus ];
-    maintainers = with maintainers; [ hrdinka govanify ];
+    license = with licenses; [
+      gpl3Plus
+      lgpl3Plus
+    ];
+    maintainers = with maintainers; [
+      hrdinka
+      govanify
+      matteopacini
+    ];
     mainProgram = "pcsx2-qt";
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" ] ++ platforms.darwin;
+    sourceProvenance =
+      lib.optional stdenv.isDarwin sourceTypes.binaryNativeCode
+      ++ lib.optional stdenv.isLinux sourceTypes.fromSource;
```

Comment:
Nice! 👍 

My first thought was to overwrite add `sourceProvenance` in the platforms specific package files, e.g. for `darwin.nix`:
```
meta = meta // {
  sourceProvenance = lib.sourceTypes.binaryNativeCode;
};
```

Not sure if there is any benefit to that, but thought I'd share…

Reply:
I've seen plenty of different approaches for this, not sure which is the best / preferred way of doing it.

Happy to change it if you think it can be improved.

Reply:
I don't think it would be an improvement per se, just a different way of doing it. Until there is a good reason to change it or a better way to do it, I'd leave it as is 🙂

Reply:
Don't really have an opinion on where to put the attribute, whatever works is fine.

However, packages that are pretty much just fetching a binary usually set `hydraPlatforms = []` because binaries don't really add value to the cache. In this case that would have to be `hydraPlatforms = lib.platforms.linux` or a different list as long as it excludes the Darwins.

Reply:
@toonn done, and thanks for the feedback!